Hestia/Hestia/frontend/EntryScreen/UIViewControllerAuth0.cs

```diff
@@ -0,0 +1,51 @@
+using Foundation;
+using System;
+using UIKit;
+using Hestia.Resources;
+using Auth0.OidcClient;
+using Hestia.backend.authentication;
+using System.Threading.Tasks;
+using IdentityModel.OidcClient;
+using System.Diagnostics;
+
+namespace Hestia
+{
+	// TODO remove if local/global is finished

```

Comment:
Especially for the last sprint submission: no TODO and commented out code. please make sure

Reply:
The functionality of this ViewController was replaced by UIViewControllerLocalGlobal. So it is gone in a8201d215aed3690f28a8903a1af3333351ae246